ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -138,3 +138,25 @@ LC:
   uses:
   - illuminance_setpoint
   - illuminance_sensor
+
+DFRT:
+  description: "Lighting Control dimmer fade rate & time"
+  is_abstract: true
+  uses:
+  - dimmer_fade_rate_setpoint
+  - dimmer_fade_time_setpoint
+
+ZLC:
+  description: "Lighting Control Zone light controls"
+  is_abstract: true
+  uses:
+  - zone_lighting_switch_command
+  - zone_lighting_switch_status
+  - zone_brightness_percentage_command
+
+LFA:
```

Comment:
Is this alarm needed? If so, what does a switch alarm mean? What happens to cause this alarm?

Reply:
upon checking device, it appears to be a general fault alarm. I will use "failed_alarm" instead.

Reply:
yes that would make it more clear. please adjust. and, if there are some common alarms that apply to a lot of devices, consider it as an abstract type of its own (that is, group multiple alarms together).